app/models/item.rb

```diff
@@ -18,4 +18,12 @@ def self.enabled_only
   def self.disabled_only
     where(enabled: false)
   end
+
+  def self.ready_to_ship
+    joins(invoices: :invoice_items)
+    .select("items.*, invoices.id AS invoice_id, invoices.created_at AS invoice_creation")
+    .where("invoice_items.status = 1")
```

Comment:
Could you call `.where(invoice_items: {status: 1})`?

Reply:
Yup, that works!